akka-docs/src/main/paradox/stream/operators/ActorSink/actorRef.md

```diff
@@ -1,11 +1,9 @@
 # ActorSink.actorRef
```

Comment:
could you also add links to these pages from stream/stream-integrations.md

Reply:
That seems done now (https://github.com/akka/akka/pull/27901/files#diff-ca4f7dce7251e3744b17219f07743226R298)

Reply:
we have a separate page for integration with actors, and I think it could be worth mentioning by linking from there too
in `stream/stream-integrations.md`

Reply:
Added https://github.com/akka/akka/pull/27901/commits/e56de0b88a96146b177da3aad06b87d58ca30218